GraphicsView/include/CGAL/Qt/camera_impl.h

```diff
@@ -36,7 +36,6 @@
 #include <CGAL/Qt/domUtils.h>
 #include <CGAL/Qt/keyFrameInterpolator.h>
 
-using namespace std;
 using namespace CGAL::qglviewer;
```

Comment:
Actually, I also think that that
```C++
using namespace CGAL::qglviewer;
```
All those `_impl` files should be in the `CGAL::qglviewer` namespace.